app/controllers/concerns/administrate/punditize.rb

```diff
@@ -1,6 +1,6 @@
-if Object.const_defined?("Pundit")
-  module Administrate
-    module Punditize
+module Administrate
+  module Punditize
+    if Object.const_defined?("Pundit")
```

Comment:
Would it be worth warning that `Pundit` is not defined so we don't just silently swallow this issue?

Reply:
As `Pundit` is not a dependency of the library, but is instead part of a configuration of the engine, I would feel uncomfortable raising a warning at this point. The implementor has made a specific choice in their application to not include `Pundit`. We shouldn't feel the need to remind them that they chose not to use it.